### ISSUE TITLE
[ FEATURE REQUEST ] - Topic refresh

### ISSUE THREAD

**Issue by Blizzardsev** [Just-Natsuki-Team/NatsukiModDev#548]
# Overview:
During the sprite rework/overhaul, it was noticed the older content of JN (for example, confessions, admissions, etc.) felt dated and somewhat out of character compared to newer topics and events where the handle on Natsuki's personality is much stronger, as they were written when the mod overhaul was just beginning.

Due to the sheer amount of topics to update sprites for, dialogue refreshes for these were skipped: we need to go back and revisit these so they are fully in line with what we expect from Natsuki!

## Must haves:
- All standard topics, starting from `talk_did_you_have_pets` to `talk_vtubers` to be revised to meet current expectations and standards for Natsuki's personality.
- All of the above topics to include use of the `extend` statement where they do not already.

## Could haves:
- Where appropriate, introduction of props can be considered.

## Is this feature request related to a problem? Please describe.
This is related to existing content.

**Comment by RaionArt** [comment]
I think it is a good idea to make all of the writing feel more consistent, but imo this should be low priority, as we are mainly lacking new content and going through so many changes will clearly take some time!
I guess reworking one topic at a time would be best and we shouldn't push together all of them for one update alone. 
Avoiding frustrating and pressure would work best here.

**Comment by Blizzardsev** [comment]
That's true - we can look at tackling this when time permits: perhaps we can look at introducing a new project (like Features, Support, Bugs) for items that need refreshing? It'd be easier to manage that way, and feel less like everything is piling up!

**Comment by Blizzardsev** [comment]
As we're looking at adding more poses very soon, I'll postpone work on this until #635 has been completed so we can get the best bang for our buck (and to save revisiting topics twice!).

**Comment by Blizzardsev** [comment]
Adding to this from #720 : where possible, we should look to introduce more dialogue options for the player, and save responses for later reactions in the topics we revisit.

**Comment by RaionArt** [comment]
Just wanted to add as #635 ist finished: Even though we have a plan to revisit certain outdated topics, we should also add the new poses to more fresh topics when needed.
